packages/gatsby-theme-api-docs/src/components/resource-method.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FullWidthContainer } from '@commercetools-docs/gatsby-theme-docs';
-import useReadResourceByResourcePath from '../hooks/use-read-resource-by-resource-path';
+import { useReadResourceByResourcePath } from '../hooks/use-read-resource-by-resource-path';
 import reportError from '../utils/report-error';
 import Method from './resource/method';
 
-const ResourceMethod = ({ apiKey, resource, method, title }) => {
+const ResourceMethodByKey = ({ apiKey, resource, method, title }) => {
```

Comment:
Is `ByKey` referencing `apiKey`?

Reply:
Yes.  Do you think it's worth changing the names to `..ByApiKey` ?